seed_reader/test/interactors/gateways/connectivity_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:connectivity/connectivity.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:seed_reader/interactors/gateways/connectivity.dart';
+import '../../mocks/mock_generator.mocks.dart';
+
+void main() {
+  group('#canConnectToApi', () {
+    test('when connectivity returns wifi then emits true', () {
+      final MockConnectivity connectivity = MockConnectivity();
+      when(connectivity.checkConnectivity()).thenAnswer(
+        (_) => Future<ConnectivityResult>.value(
+          ConnectivityResult.wifi,
+        ),
+      );
+
+      expect(canConnectToApi(connectivity), completion(true));
+    });
+    test('when connectivity returns none then emits false', () {
+      final MockConnectivity connectivity = MockConnectivity();
+      when(connectivity.checkConnectivity()).thenAnswer(
+        (_) => Future<ConnectivityResult>.value(
+          ConnectivityResult.none,
+        ),
+      );
+
+      expect(canConnectToApi(connectivity), completion(false));
+    });
+    test('when connectivity returns mobile then emits false', () {
```

Comment:
chore: it feels like mobile should emit true? When I see `canConnectToApi`, having a mobile connection seems like a prime `yes` in a real-world scenario.